pepper-apis/build-utils/gcloud-pubsub-emulator/Dockerfile

```diff
@@ -0,0 +1,29 @@
+# Adapted from https://github.com/marcelcorso/gcloud-pubsub-emulator
+FROM golang:alpine as builder
+
+RUN apk update && apk upgrade && apk add --no-cache curl git
+
+RUN curl -s https://raw.githubusercontent.com/eficode/wait-for/master/wait-for -o /usr/bin/wait-for
+RUN chmod +x /usr/bin/wait-for
+
+RUN go get github.com/prep/pubsubc
+
+###############################################################################
+
+FROM google/cloud-sdk:alpine
+
+RUN apk --no-cache add \
+    bash \
+    openjdk8-jre \
+    netcat-openbsd && \
+    gcloud components install beta pubsub-emulator
+
+COPY --from=builder /usr/bin/wait-for /usr/bin
+COPY --from=builder /go/bin/pubsubc   /usr/bin
+COPY                run.sh            /run.sh
+
+ENV PORT 8442
+
+EXPOSE $PORT
+
+CMD /run.sh $PORT
```

Comment:
Leaving out the gcloud beta tools (which include the emulator) saved a lot of time in start up by keeping base build image thin. But we still need a pubsub emulator to run tests.
Turns out starting pubsub emulator in separate image is much faster.